ouroboros-consensus/test-consensus/Test/Consensus/Mempool.hs

```diff
@@ -318,11 +318,14 @@ ppTestSetup :: TestSetup -> String
 ppTestSetup TestSetup { testInitialTxs
                       , testMempoolCap = MempoolCapacityBytes mpCap
                       } = unlines $
-    ["Initial contents of the Mempool:"] <>
-    (map condense testInitialTxs)        <>
-    ["Mempool capacity:"]                <>
+    ["Initial contents of the Mempool:"]  <>
+    (map ppTestTxWithHash testInitialTxs) <>
+    ["Mempool capacity:"]                 <>
     [condense mpCap]
 
+ppTestTxWithHash :: TestTx -> String
+ppTestTxWithHash x = condense (hash (simpleGenTx x) :: Hash ShortHash Tx, x)
```

Comment:
I'm assuming you're using `ShortHash` here instead of `MD5` for readability, but then it deserves a comment mentioning that `ShortHash` is just the first four bytes of the `MD5` hash. If that weren't the case, printing the hashes wouldn't be useful.

Reply:
No, I meant to use MD5 for precision. Just neglected to update the snippet when copying from my debugging worktree.